action.yaml

```diff
@@ -23,7 +23,7 @@ inputs:
   display-workflow-run-url:
     description: 'Get the URL of the triggered workflow and display it in logs (useful to follow the progress of the triggered workflow)'
     required: false
-    default: true
```

Comment:
Why did you add quotes ? It should be a boolean value

Reply:
Should this line be changed? https://github.com/aurelien-baudet/workflow-dispatch/blob/00dba38060567621bfc0f88e7fc19a4932f4312f/src/utils.ts#L44

Reply:
Not really, why ?

Reply:
> It should be a boolean value

Looking at the [schema](https://json.schemastore.org/github-action.json), the type of `default` is `string`. However, I don't think it will actually change any behaviour as GHA likely coerces it into a string anyway. This [StackOverflow answer](https://stackoverflow.com/questions/19109912/yaml-do-i-need-quotes-for-strings-in-yaml) may also help explain.

It may be worth adding the quotes if only to avoid your editor showing a syntax error, but it should probably then use single-quotes to be consistent with the rest of the file.

> Should this line be changed?

I also don't think this line should be changed. Whether the input is processed as a string or boolean is dependent on whether `core.getInput` or `core.getBooleanInput` is used in the script: https://www.npmjs.com/package/@actions/core#inputsoutputs